<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>HostProfiles</class>
  <widget class="QWidget" name="HostProfiles" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>222</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item rowspan="1" row="0" column="0" colspan="3" >
        <widget class="K3ListView" name="hostListView" >
          <property name="selectionMode" stdset="0" >
            <enum>Multi</enum>
          </property>
          <property name="allColumnsShowFocus" >
            <bool>true</bool>
          </property>
          <property name="fullWidth" >
            <bool>true</bool>
          </property>
          <property name="itemsMovable" >
            <bool>false</bool>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>This list shows all hosts that you have visited and a summary of your settings for them. If you want to reset the setting for a host, you can delete it using the buttons below. When you connect again you can then re-configure them.</string>
          </property>
          <column>
            <property name="text" >
              <string>Host</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
          <column>
            <property name="text" >
              <string>Type</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
          <column>
            <property name="text" >
              <string>Settings</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
        </widget>
      </item>
      <item row="1" column="0" >
        <spacer name="spacer3" >
          <property name="sizeHint" >
            <size>
              <width>117</width>
              <height>21</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="1" >
        <widget class="QPushButton" name="removeHostButton" >
          <property name="text" >
            <string>&amp;Remove Selected Host</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Deletes the hosts that you have selected in the list above.</string>
          </property>
        </widget>
      </item>
      <item row="1" column="2" >
        <widget class="QPushButton" name="removeAllButton" >
          <property name="text" >
            <string>Remove &amp;All Hosts</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Removes all hosts from the list.</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header>k3listview.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >hostpreferences.h</include>
  </includes>
</ui>
